vita3k/renderer/src/vulkan/surface_cache.cpp

```diff
@@ -780,7 +780,7 @@ std::optional<TextureLookupResult> VKSurfaceCache::retrieve_depth_stencil_as_tex
     if (cached_info.multisample_mode == SCE_GXM_MULTISAMPLE_4X)
         height /= 2;
 
-    const bool is_stencil = cached_info.surface.stencil_data.address() == address;
+    const bool is_stencil = can_be_stencil;
```

Comment:
no can juste remove "is_stencil" and using directly "can_be_stencil" ?

Reply:
It doesn't really have the same meaning, there is a can_be_stencil and can_be_color declaration at the beginning meaning the texture may be a stencil or a color and here is_stencil means for sure it is a stencil.

Reply:
Using a variable named can_be_stencil below can lead to a misunderstanding and using a variable named is_stencil above can also lead to a misunderstanding.

Reply:
I see 🤔